docker/defaults.inc.sh

```diff
@@ -33,3 +33,12 @@ PHP_EXTENSIONS=""
 # crond is the only allowed cronjob handler until we got a solution for jenkins too
 # crond might be dropped then, so please don't rely on "crond" in your shop flavour!
 CRONJOB_HANDLER="crond"
+
+#  Codeception tests
+TEST_RUN_CMD="vendor/bin/codecept run"
+TESTS_IGNORED_GROUPS="-x CheckoutAvailabilityCest \
```

Comment:
Yes, I agree.. But it was a suggestion made by Tony, actually. To put everything in the base image.